src/main/java/de/rub/nds/modifiablevariable/util/ArrayConverter.java

```diff
@@ -13,43 +13,89 @@
 import java.util.List;
 
 /** A utility class to handle arrays and array conversions */
-public class ArrayConverter {
+public final class ArrayConverter {
+
+    private ArrayConverter() {
+        super();
+    }
 
     /**
      * Takes a long value and converts it to 8 bytes (needed for example to convert SQN numbers in
      * TLS records)
      *
-     * @param l long value
+     * @param value long value
      * @return long represented by 8 bytes
      */
-    public static byte[] longToUint64Bytes(long l) {
+    public static byte[] longToEightBytes(long value) {
```

Comment:
why this rename?

Reply:
Because the name is wrong and more confusing in my opinion. It is not uint64. The interpretation of the bytes depends on the context. But since long is signed, the result would be more like int64.

But I can revert them back if you would like

Reply:
reverted